tests/console/sysstat.pm

```diff
@@ -24,13 +25,17 @@ sub run {
     systemctl 'restart sysstat.service';
 
     #compare todays date with the date on the file generated when starting the service/systemd unit:
-    validate_script_output "if [ `date +'%Y%m%d'` -eq `ls -1 /var/log/sa/ | cut -c 3-` ]; then echo date_matched; fi", sub { /date_matched/ };
+    if (!is_sle('=12-SP4')) {
```

Comment:
What's about flipping if and else to avoid negation, which will improve performance + readability of the code?

Reply:
Fixed.